config/checkstyle/checkstyle.xml

```diff
@@ -425,6 +425,7 @@
       <property name="minLineCount" value="1"/>
       <property name="allowMissingPropertyJavadoc" value="true"/>
       <property name="ignoreMethodNamesRegex" value="(set.*|get.*)"/>
+      <property name="tokens" value="METHOD_DEF, ANNOTATION_FIELD_DEF"/>
```

Comment:
Some constructors may not need a javadoc due to being self-explanatory, but this is not always the case, especially when there there are multiple constructors. Also, our coding standard doesn't allow omitting javadocs for constructors.
So, let's not do this.
Besides, it's not related to this PR.